mne_bids/write.py

```diff
@@ -952,3 +949,117 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
             sh.copyfile(value, marker_fname)
 
     return output_path
+
+
+def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
+               raw=None, trans=None, overwrite=False, verbose=False):
+    """Put anatomical MRI data into a BIDS format.
+
+    Given a BIDS directory and a T1 weighted MRI scan for a certain subject,
+    format the MRI scan to be in BIDS format and put it into the correct
+    location in the bids_dir. If a transformation matrix is supplied, a
+    sidecar JSON file will be written for the T1 weighted data.
+
+    Parameters
+    ----------
+    bids_root : str
+        Path to root of the BIDS folder
+    subject : str
+        Subject label as in 'sub-<label>', for example: '01'
+    t1w : str
+        Path to a T1 weighted MRI scan of the subject. Must be in .nii,
+        .nii.gz, or .mgz format. If in .mgz, it will be converted to .nii.gz
+    session : str | None
+        The session for `t1w`. Corresponds to "ses"
+    acquisition: str | None
+        The acquisition parameters for `t1w`. Corresponds to "acq"
+    raw : instance of Raw | None
+        The raw data of `subject` corresponding to `t1w`. If `raw` is None,
+        `trans` has to be None as well
+    trans : instance of mne.transforms.Transform | None
+        The transformation matrix from HEAD coordinates to MRI coordinates.
+        If None, no sidecar JSON file will be written for `t1w`
+    overwrite : bool
+        Whether to overwrite existing files or data in files.
+        Defaults to False.
+        If overwrite is True, any existing files with the same BIDS parameters
+        will be overwritten with the exception of the `participants.tsv` and
+        `scans.tsv` files. For these files, parts of pre-existing data that
+        match the current data will be replaced.
+        If overwrite is False, no existing data will be overwritten or
+        replaced.
+    verbose : bool
+        If verbose is True, this will print a snippet of the sidecar files. If
+        False, no content will be printed.
+
+    Returns
+    -------
+    anat_dir : str
+        Path to the anatomical scan in the `bids_dir`
+
+    """
+    try:  # pragma: no cover
+        import nibabel as nib
+    except ImportError:  # pragma: no cover
+        raise ImportError('This function requires nibabel.')
```

Comment:
can be refactored into check_nibabel? doesn't MNE have such a function that you could reuse?